tools/prerequisites/k8s-setup.sh

```diff
@@ -107,6 +107,47 @@ configure_container() {
     done
 }
 
+add_containerd_proxy() {
```

Comment:
So we assume that the users have configured all types of proxy? eg. what will happen if we configure the containerd service with no_proxy=""

Reply:
If we configure the containerd service with
```
[Service]
...
Environment="NO_PROXY="
```
the it will not use no proxy for containerd.
And if users haven't configured any types of proxy, the containerd proxy will not be configured.